src/renderer/ducks/register/slice.ts

```diff
@@ -0,0 +1,54 @@
+import {createSlice, PayloadAction} from "@reduxjs/toolkit";
+import {networks} from "../../services/eth2/networks";
+
+export interface IRegisterState {
+    signingMnemonic: string,
+    signingVerification: boolean,
+    signingKey: string,
+    signingKeyPath: string,
+    withdrawalKey: string,
+    network: string;
+}
+
+const initialState: IRegisterState = {
+    signingMnemonic: "",
+    signingVerification: false,
+    signingKey: "",
+    signingKeyPath: "",
+    withdrawalKey: "",
+    network: networks[0]?.networkName ?? "unknown",
+};
+
+export const registerSlice = createSlice({
+    name: "register",
+    initialState,
+    reducers: {
+        storeSigningMnemonic: (state, action: PayloadAction<string>): void => {
+            state.signingMnemonic = action.payload;
+        },
+        storeSigningVerificationStatus: (state, action: PayloadAction<boolean>): void => {
+            state.signingVerification = action.payload;
+        },
+        storeSigningKey: (state, action: PayloadAction<string>): void => {
+            state.signingKey = action.payload;
+        },
+        storeValidatorKeys: {
+            reducer: (
+                state,
+                action: PayloadAction<{signingKey: string; withdrawalKey: string; signingKeyPath: string;}>
+            ): void => {
+                state.signingKey = action.payload.signingKey;
+                state.withdrawalKey = action.payload.withdrawalKey;
+                state.signingKeyPath = action.payload.signingKeyPath;
+            },
+            prepare: (signingKey: string, withdrawalKey: string, signingKeyPath: string):
+            {payload: {signingKey: string; withdrawalKey: string; signingKeyPath: string;}} => ({
```

Comment:
export `signingKey: string; withdrawalKey: string; signingKeyPath: string;` to interface as it's reused a lot here